sims.code-workspace

```diff
@@ -128,6 +128,9 @@
     },
     "[sql]": {
       "editor.defaultFormatter": "adpyke.vscode-sql-formatter"
+    },
+    "[vue]": {
```

Comment:
Was this not already covered on line 80?
Was this a solution for the "auto-save" issue that you were facing?

Reply:
We can keep it if it is a configuration to share between the devs.
Is the config not overridden by some user settings level?
To allow the PR to move ahead I would revert it till we are sure.